packages/snaps-cli/scripts/createInitTemplate.js

```diff
@@ -29,7 +29,7 @@ async function createInitTemplate() {
     `${JSON.stringify(
       {
         html: html.toString(),
-        source: js.toString(),
+        source: js.toString().replace(/\r\n/gu, '\n'),
```

Comment:
Do we need to have the check for new line if we're ultimately replacing with newline? Can it be just `.replace(/\r/gu, '\n')` ?

Reply:
Nevermind! for some reason I thought it was or 😅

Reply:
Good question. Whenever I do regex manipulations of strings in this manner, I try to preclude undesired modifications to the greatest extent possible. This `.replace` call is meant to replace CRLF line breaks with LF line breaks and nothing else, so we try to do just that.

Reply:
I added comments to explain this, and simplified the call.